Build error

I'm on expo v45 and get this when building the dev client through eas

```bash
[stderr] 
FAILURE: Build failed with an exception.
[stderr] 
* What went wrong:
[stderr] 
Execution failed for task ':app:mergeDebugNativeLibs'.
[stderr] 
> A failure occurred while executing com.android.build.gradle.internal.tasks.MergeNativeLibsTask$MergeNativeLibsTaskWorkAction
[stderr] 
   > 2 files found with path 'lib/arm64-v8a/libjsinspector.so' from inputs:
[stderr] 
      - /home/expo/workingdir/build/node_modules/react-native-v8/android/build/intermediates/library_jni/debug/jni/arm64-v8a/libjsinspector.so
[stderr] 
      - /home/expo/.gradle/caches/transforms-3/7d9d92dc8ec1ba2e45aff2ecbb549550/transformed/jetified-react-native-0.68.2/jni/arm64-v8a/libjsinspector.so
[stderr] 
     If you are using jniLibs and CMake IMPORTED targets, see
[stderr] 
     https://developer.android.com/r/tools/jniLibs-vs-imported-targets
[stderr] 
```

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -183,6 +183,7 @@ android {
       "**/libfolly_json.so",
       "**/libglog.so",
       "**/libreactnativejni.so",
+      "**/libjsinspector.so",
     ]
     pickFirst "**/libv8android.so"
   }

